server/src/main/java/org/apache/druid/server/coordinator/duty/KillUnusedSegments.java

```diff
@@ -270,6 +296,12 @@ private int getAvailableKillTaskSlots(int killTaskCapacity, int numActiveKillTas
     );
   }
 
+  @VisibleForTesting
```

Comment:
Nit:
Can we do something other than making this method visible?
We could just test the `run` method itself.

Druid code is already flooded with this annotation and we seem to keep adding more. It really defeats the point of having public and private methods. In some cases, where the code is really involved, it probably helps to test out individual methods. But this one seems to be a simple case that can be tested in other ways.

Reply:
The annotation isn't needed anymore with the updated logic as the duty will run at least once.